interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -595,8 +596,8 @@ public function getAllergies($pid, $encounter)
                 }
 
                 if ($row['reaction']) {
-                    $reaction_text = \Carecoordination\Model\CarecoordinationTable::getListTitle($row['reaction'], 'reaction', '');
-                    $reaction_code = \Carecoordination\Model\CarecoordinationTable::getCodes($row['reaction'], 'reaction');
+                    $reaction_text = getListTitle($row['reaction'], 'reaction', '');
+                    $reaction_code = getCodes($row['reaction'], 'reaction');
```

Comment:
these don't look right. Should they look like:
```
$reaction_text = CarecoordinationTable::getListTitle($row['reaction'], 'reaction', '');
$reaction_code = CarecoordinationTable::getCodes($row['reaction'], 'reaction');
```

Reply:
Thanks for the review @bradymiller 😄